src/html-conventions-lib/connectors-with-same-name.xsl

```diff
@@ -0,0 +1,118 @@
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+    xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dct="http://purl.org/dc/terms/"
+    xmlns:skos="http://www.w3.org/2004/02/skos/core#"
+    xmlns:f="http://https://github.com/costezki/model2owl#" version="3.0">
+
+    <xsl:import href="../common/checkers.xsl"/>
+    <xsl:import href="utils-html-conventions.xsl"/>
+
+    <xd:doc>
+        <xd:desc>Applying the checkers to a group of connectors with same name</xd:desc>
+    </xd:doc>
+
+    <xsl:template name="connectorsWithSameName">
+        <xsl:variable name="root" select="root()"/>
+        <xsl:variable name="distinctNames" select="f:getDistinctConnectorsNames($root)"/>
+        <h1>Connectors with the same name</h1>
+        <xsl:for-each select="$distinctNames">
+            <xsl:if test="fn:count(f:getConnectorByName(., $root)) > 1">
+
+                <xsl:variable name="connectorsChecks" as="item()*">
+                    <xsl:call-template name="multiplicityForConnectorsWithSameName">
+                        <xsl:with-param name="connectorName" select="."/>
+                        <xsl:with-param name="root" select="$root"/>
+                    </xsl:call-template>
+                    <xsl:call-template name="definitionForConnectorsWithSameName">
+                        <xsl:with-param name="connectorName" select="."/>
+                        <xsl:with-param name="root" select="$root"/>
+                    </xsl:call-template>
+                </xsl:variable>
+                <xsl:if test="boolean($connectorsChecks)">
+                    <dt>
+                        <xsl:value-of select="."/>
+                    </dt>
+                    <xsl:copy-of select="$connectorsChecks"/>
+                </xsl:if>
+            </xsl:if>
+        </xsl:for-each>
+    </xsl:template>
+
+
+
+
+
+    <xd:doc>
+        <xd:desc>Check the multiplicity values from a group of connectors with same name</xd:desc>
+        <xd:param name="connectorName"/>
+        <xd:param name="root"/>
+    </xd:doc>
+    <xsl:template name="multiplicityForConnectorsWithSameName">
+        <xsl:param name="connectorName"/>
+        <xsl:param name="root"/>
+        <xsl:variable name="connectorsWithSameName"
+            select="f:getConnectorByName($connectorName, $root)"/>
+        <xsl:variable name="multiplicityValues"
+            select="$connectorsWithSameName/*[role/@name = $connectorName]/type/@multiplicity"/>
+        <xsl:variable name="allConnectorsHaveMultiplicityValue"
+            select="fn:count($connectorsWithSameName) = fn:count($multiplicityValues)"/>
+        <xsl:sequence
+            select="
+                if (f:areStringsEqual($multiplicityValues) and $allConnectorsHaveMultiplicityValue) then
+                    ()
+                else
+                    f:generateHtmlWarning('The multiplicity value used is not equal for all the connectors with this name')"
```

Comment:
`When a property is reused in multiple contexts, the multiplicity is expected to be the same. Please check the nomenclature above for a summary.`